snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -1,56 +1,28 @@
-## http://snmplabs.com/pysnmp/docs/api-reference.html#user-based
+##
+## NOTE: See SNMP integration setup guide and examples at https://docs.datadoghq.com/network_monitoring/devices/setup
 
 init_config:
 
   ## @param loader - string - optional - default: python
   ## Check loader to use for all instances. Available loaders:
-  ## - core: (recommended) will use new corecheck SNMP integration
+  ## - core: (recommended) will use new corecheck SNMP integration (faster and lower mem usage)
   ## - python: will use legacy python SNMP integration
   #
   loader: core
 
-  ## @param mibs_folder - string - optional
-  ## Specify an additional folder for your custom mib files (python format).
-  #
-  # mibs_folder: /path/to/your/mibs/folder
-
-  ## @param ignore_nonincreasing_oid - boolean - optional - default: false
-  ## Set to true to ignore non increasing oid.
-  ## See https://github.com/etingof/pysnmp/blob/master/docs/source/faq/oids-not-increasing.rst
-  #
-  # ignore_nonincreasing_oid: false
-
   ## @param oid_batch_size - integer - optional - default: 10 (python), 60 (core)
   ## The number of OIDs handled by each batch. Increasing this number improves performance but
   ## uses more resources.
   #
   # oid_batch_size: 10
 
-  ## @param optimize_mib_memory_usage - boolean - optional - default: false
-  ## **Experimental** - Setting this flag to true attempts to optimize memory usage related to
-  ## MIB information. This shares internal MIB data across instances.
-  ## Note: The absence of impact on MIB resolution for this strategy is not guaranteed yet.
-  #
-  # optimize_mib_memory_usage: false
-
-  ## @param refresh_oids_cache_interval - integer - optional - default: 0
-  ## (beta feature) Set this option to enable caching of OIDs. The value is the number of seconds before the
-  ## the cache is refreshed.
-  ## This option works best when `oid_batch_size` is set to 128 or 256.
-  ## This is useful to improve performance when fetching metrics from large tables.
-  ## When OID caching is enabled, table row OIDs are reused. This means that the table can be fetched
-  ## using batched GET queries, as opposed to one GETNEXT query per row in the table.
-  ## The caching is disabled if the interval is set to `0` (default).
-  ##
-  ## Caveat: If some tables rows are dynamic, new rows will only be fetched at every refresh interval.
-  ##         - This means new OIDs will only be picked up when OIDs are refreshed according to the refresh interval.
-  ##         - With auto discovery, if the same IP is assigned to a different device with different OIDs,
-  ##         the OIDs will be refreshed only according to the refresh interval.
+  ## @param collect_device_metadata - boolean - optional - default: false
+  ## Enable device metadata collection
+  ## Only available using corecheck SNMP integration with `loader: core` config.
   #
-  # refresh_oids_cache_interval: 3600
+  # collect_device_metadata: true
```

Comment:
Can we remove the option in 7.29? The underlying pipeline is not ready yet to receive payload.

Reply:
@gmarabout Sure, let's do it in this revert PR since it's not the purpose of this PR: https://github.com/DataDog/integrations-core/pull/9439